mautic/mautic
# Set timezone from browser first instead of guess it

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
We noticed the timezone set from the query is guessing timezone, even If the timezone is passed in the query directly. 
We fixed it



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Enable public update able for timezone field

![image](https://github.com/mautic/mautic/assets/462477/f1cff543-388e-4ce5-b181-eed439c9657e)

3. Add tracking pixel to page
4. Check If timezone was set from browser correctly
